packages/agreement-process/test/testArchiveAgreement.ts

```diff
@@ -0,0 +1,165 @@
+import { fail } from "assert";
+import { v4 as uuidv4 } from "uuid";
+import {
+  Agreement,
+  AgreementId,
+  AgreementUpdatedV1,
+  AgreementV1,
+  EServiceId,
+  TenantId,
+  agreementState,
+  generateId,
+  protobufDecoder,
+} from "pagopa-interop-models";
+import { describe, expect, it, vi } from "vitest";
+import {
+  StoredEvent,
+  getMockAgreement,
+  getRandomAuthData,
+  randomArrayItem,
+} from "pagopa-interop-commons-test/index.js";
+import { toAgreementV1 } from "../src/model/domain/toEvent.js";
+import {
+  agreementNotFound,
+  agreementNotInExpectedState,
+  operationNotAllowed,
+} from "../src/model/domain/errors.js";
+import { agreementArchivableStates } from "../src/model/domain/validators.js";
+import { addOneAgreement, readLastAgreementEvent } from "./utils.js";
+import {
+  agreementService,
+  agreements,
+  postgresDB,
+} from "./agreementService.integration.test.js";
+
+export function testArchiveAgreement(): void {
```

Comment:
After you rebased I made another slight change in #377 - returning the `describe` result to make sure it's actually equivalent to calling describe in the main tests file: https://github.com/pagopa/interop-be-monorepo/pull/377/commits/f6322ea9fa07070dd84d473a29da930e91637b1c

Could you integrate it here as well? 🙏